apps/kyb-app/src/common/utils/request.ts

```diff
@@ -1,13 +1,57 @@
 import { getAccessToken } from '@app/helpers/get-access-token.helper';
-import ky from 'ky';
+import ky, { HTTPError } from 'ky';
+import * as Sentry from '@sentry/react';
 
-export const request = ky.extend({
-  prefixUrl: import.meta.env.VITE_API_URL as string,
+export const request = ky.create({
+  prefixUrl: import.meta.env.VITE_API_URL || `${window.location.origin}/api/v1/`,
+  retry: {
+    limit: 1,
+    statusCodes: [500, 408, 404, 404, 403, 401],
+    methods: ['get'],
+  },
+  timeout: 30_000,
   hooks: {
     beforeRequest: [
       request => {
         request.headers.set('Authorization', `Bearer ${getAccessToken()}`);
       },
     ],
+    beforeError: [
+      // TODO: catch Workflowsdk API Plugin errors as well
```

Comment:
Task added ? will be nice to add the task number if you want to keep the TODO

Reply:
K

Reply:
Task has been added with informative explanation and also related issue than it will also be easy to discover this PR as well 
https://linear.app/ballerine/issue/BAL-740/web-app-collection-flow-catch-api-plugin-errors